specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json

```diff
@@ -1695,7 +1783,7 @@
           "type": "string",
           "description": "The reason for approval or rejection."
         },
-        "actionRequired": {
+        "actionsRequired": {
```

Comment:
What did the service actually return? If `actionRequired` was never returned, this is probably a fine change to make, but @johanste and the rest of the REST review board needs to make the call.

Reply:
The service always returned actionsRequired. This was a bug in the swagger